tests/sklearn/test_sklearn_autolog.py

```diff
@@ -859,24 +858,6 @@ def test_autolog_does_not_throw_when_parameter_search_logging_fails(failing_spec
         mock_func.assert_called_once()
 
 
-@pytest.mark.usefixtures(test_mode_off.__name__)
-@pytest.mark.parametrize(
-    "func_to_fail",
-    ["mlflow.log_params", "mlflow.log_metric", "mlflow.set_tags", "mlflow.sklearn.log_model"],
-)
-def test_autolog_does_not_throw_when_mlflow_logging_fails(func_to_fail):
```

Comment:
This test case is no longer needed; MLflow's autologging safety tests are meant to verify that errors in MLflow's autologging implementations do not disrupt training (see https://github.com/mlflow/mlflow/blob/master/tests/autologging/test_autologging_safety_unit.py)